.github/workflows/custom-event-action.yml

```diff
@@ -37,4 +46,4 @@ jobs:
       env:
         OKAY_API_TOKEN: ${{ secrets.OKAY_API_TOKEN }}
       run: |
-        curl -X POST -H "Authorization: Bearer $OKAY_API_TOKEN" -H 'Content-Type: application/json' --data "{\"event\": \"custom\",  \"uniqueKey\": [\"event_unique_id\"], \"customEventName\": \"${{ inputs.event }}\", \"timestamp\": \"$(date +%Y-%m-%dT%H:%M:%S%z)\", \"identity\": {\"user\": \"${{ inputs.user }}\", \"type\": \"sourceControlLogin\"}, \"properties\": {\"event_unique_id\": \"${{ inputs.event_unique_id }}\", \"${{ inputs.metric1_name }}\": {\"type\": \"durationMs\", \"value\": ${{ inputs.metric1_duration_ms }}}, \"${{ inputs.metric2_name }}\": {\"type\": \"number\", \"value\": ${{ inputs.metric2_number }}}, \"${{ inputs.metric3_name }}\": {\"type\": \"count\", \"value\": ${{ inputs.metric3_count }}} }}" https://app.okayhq.com/api/events/v1
\ No newline at end of file
+        curl -X POST -H "Authorization: Bearer $OKAY_API_TOKEN" -H 'Content-Type: application/json' --data "{\"event\": \"custom\",  \"uniqueKey\": [\"event_unique_id\"], \"customEventName\": \"${{ inputs.event }}\", \"timestamp\": \"$(date +%Y-%m-%dT%H:%M:%S%z)\", \"identity\": {\"user\": \"${{ inputs.user }}\", \"type\": \"sourceControlLogin\"},\"labels\":[\"${{ inputs.label1 }}\",\"${{ inputs.label2 }}\"], \"properties\": {\"event_unique_id\": \"${{ inputs.event_unique_id }}\", \"okay.url\": \"${{ inputs.url }}\", \"${{ inputs.metric1_name }}\": {\"type\": \"durationMs\", \"value\": ${{ inputs.metric1_duration_ms }}}, \"${{ inputs.metric2_name }}\": {\"type\": \"number\", \"value\": ${{ inputs.metric2_number }}}, \"${{ inputs.metric3_name }}\": {\"type\": \"count\", \"value\": ${{ inputs.metric3_count }}} }}" https://app.okayhq.com/api/events/v1
```

Comment:
just made a note to add special handling for `okay.url` in the general solution.